app.js

```diff
@@ -0,0 +1,110 @@
+// import fetch library to fetch the data from API
+const fetch = require('node-fetch');
+
+
+
+
+// arraow function to fetch for currency
+const getCountries = async () => {
+    const response = await fetch("https://gist.githubusercontent.com/tiagodealmeida/0b97ccf117252d742dddf098bc6cc58a/raw/f621703926fc13be4f618fb4a058d0454177cceb/countries.json")
+    const jsonRes = await response.json()
+    return jsonRes
+}
+
+
+// arraow function to fetch for country
+const getCurrency = async () => {
+    const response = await fetch("https://gist.githubusercontent.com/ksafranski/2973986/raw/5fda5e87189b066e11c1bf80bbfbecb556cf2cc1/Common-Currency.json")
+    const jsonRes = await response.json()
+    return jsonRes
+}
```

Comment:
here, there is a duplicated code, please read about DRY principle and apply it here